core/templates/pages/exploration-player-page/services/exploration-engine.service.ts

```diff
@@ -541,6 +543,23 @@ export class ExplorationEngineService {
     questionHtml = questionHtml + this._getRandomSuffix();
     nextInteractionHtml = nextInteractionHtml + this._getRandomSuffix();
 
+    // Check if the answer is only misspelled and close
+    // to the correct answer. Only to be checked if the
+    // interaction is TextInput.
+    let oldInteractionId = oldState.interaction.id;
+
+    // Work => let catchMisspellingsFeatOn =
+    // oldStateCard.getInteraction().customizationArgs.
+
+    if (oldInteractionId === 'TextInput' && onSameCard) {
```

Comment:
Use constant for `"TextInput"`

Reply:
Actually, this feature needs to be enabled when catchMisspellings will be true. The work for the customization_arg is in another large PR that is under review currently.

Reply:
Done.